clouds/databricks/README.md

```diff
@@ -0,0 +1,161 @@
+# CARTO Analytics Toolbox
+
+[![CI](https://github.com/cartodb/analytics-toolbox-databricks/actions/workflows/ci.yml/badge.svg)](https://github.com/cartodb/analytics-toolbox-databricks/actions/workflows/ci.yml)
+[![Maven Badge](https://img.shields.io/maven-central/v/com.carto.analyticstoolbox/core_2.12?color=blue)](https://search.maven.org/search?q=g:com.carto.analyticstoolbox%20and%20core)
+[![Snapshots Badge](https://img.shields.io/nexus/s/https/s01.oss.sonatype.org/com.carto.analyticstoolbox/core_2.12)](https://s01.oss.sonatype.org/content/repositories/snapshots/com/carto/analyticstoolbox/core_2.12/)
+
+CARTO Analytics Toolbox for Databricks provides geospatial functionality leveraging the GeoMesa SparkSQL capabilities. It implements Spatial Hive UDFs and consists of the following modules:
+
+* `core` with Hive GIS UDFs (depends on [GeoMesa](https://github.com/locationtech/geomesa), [GeoTrellis](https://github.com/locationtech/geotrellis), and [Hiveless](https://github.com/azavea/hiveless))
+
+## Quick Start
+
+```scala
+resolvers ++= Seq(
+  // for snapshot artifacts only
+  "s01-oss-sonatype" at "https://s01.oss.sonatype.org/content/repositories/snapshots"
+)
+
+libraryDependencies ++= Seq(
+  "com.carto.analyticstoolbox" %% "core" % "<latest version>"
+)
+```
+
+## Supported GIS functions (core)
+
+```sql
+CREATE OR REPLACE FUNCTION st_geometryFromText as 'com.carto.analyticstoolbox.core.ST_GeomFromWKT';
+CREATE OR REPLACE FUNCTION st_intersects as 'com.carto.analyticstoolbox.core.ST_Intersects';
+CREATE OR REPLACE FUNCTION st_simplify as 'com.carto.analyticstoolbox.core.ST_Simplify';
+ -- ...and more
+```
+
+The full list of supported functions can be found [here](./core/sql/createUDFs.sql).
+
+## Spatial Query Optimizations
+
+There are two types of supported optimizations: `ST_Intersects` and `ST_Contains`, which allow Spark to push down predicates when possible.
+
+To enable optimizations:
+
+```scala
+com.carto.analyticstoolbox.spark.sql.rules.SpatialFilterPushdownRules
+
+val spark: SparkSession = ???
```

Comment:
This part of the doc has to be completed? or the user has to include his config?

Reply:
This is when the user creates a spark session. It depends on where is the spark session being created.
For instance the spark session in the databricks notebooks is created automatically and you can access it with the variable spark. (no need for this line in that case)